docs/UserGuide.md

```diff
@@ -308,18 +308,18 @@ to the current date by sorting the list according to the students' Mastery Check
 Format: `sort CRITERIA/ORDER`
 
 * `CRITERIA` can be
-    * `n` (name)
-    * `t` ([Telegram handle](#glossary))
-    * `c` (consultation)
-    * `m` ([Mastery Check](#glossary))
-    * `ra1` ([Reading Assessment 1](#glossary))
-    * `ra2` ([Reading Assessment 2](#glossary))
-    * `pa` ([Practical Assessment](#glossary))
-    * `mt` ([Midterm Test](#glossary))
-    * `ft` ([Final Examination](#glossary))
+    * Name - `n`
```

Comment:
Can add back the hyperlinks to the glossary

Reply:
Ok! added them back